src/Commando/Option.php

```diff
@@ -243,6 +254,19 @@ public function map($value)
         return call_user_func($this->map, $value);
     }
 
+    /**
+     * @param mixed $value
+     * @return Option
+     */
+    public function reduce($accumulator, $value)
```

Comment:
might be nice to include an optional initial value

Reply:
Agreed. At the time of writing I assumed this would be another use of `default()`, and it will get the default set using `default()` to seed the initial value of the accumulator.

I  could ad an optional `$default` or `$seed` param on `Command::_reduce`, but I think I would implement it in a way that would just set the default value using `default()` anyway.

What do you think?